Formula/ldid.rb

```diff
@@ -17,7 +17,7 @@ class Ldid < Formula
   depends_on "openssl"
 
   def install
-    inreplace "./make.sh", /^.*\/Applications\/Xcode-5.1.1.app.*/, ""
+    inreplace "./make.sh", %r{/^.*\/Applications\/Xcode-5.1.1.app.*/}, ""
```

Comment:
I think you've done the same thing in each of the regex changes where nudged. Shout if you need any help rewriting those to the above sort of style.